pyo3-derive-backend/src/pyclass.rs

```diff
@@ -90,6 +90,17 @@ impl PyClassArgs {
                 self.freelist = Some(syn::Expr::clone(right));
             }
             "name" => match &**right {
+                syn::Expr::Group(exp_grp) => {
+                    if let syn::Expr::Path(exp) = *(exp_grp.expr).clone() {
+                        if exp.path.segments.len() == 1 {
+                            self.name = Some(exp.clone().into());
+                        } else {
+                            expected!("Must be Name not Path::Name")
+                        }
+                    } else {
+                        expected!("type name (e.g., Name), found: {:?}", exp_grp)
+                    }
+                }
```

Comment:
`expected` macro does not accept formatting args. Just `type name` would be sufficient.
And this can be simplified like
```suggestion
                // For macro args(e.g., $name)
                syn::Expr::Group(exp_grp) => match &*exp_grp.expr {
                    syn::Expr::Path(exp) if exp.path.segments.len() == 1 => {
                        self.name = Some(exp.clone().into());
                    }
                    _ => expected!("type name (e.g., Name)"),
                },
```